dist/svg.js

```diff
@@ -1694,20 +1704,53 @@ SVG.FX = SVG.invent({
 
       // convert current time to position
       if(!ignoreTime) this.pos = this.timeToPos(+new Date)
+      else {
+        // If the method at was called with a position above 1, we set it back to 1
+        if(this.pos > 1) this.pos = 1
```

Comment:
Why is this done here? Or better: Why is the other check done outside of the else? I rather would have 2 times the same `if`, when this enables us to group the both checks together